src/clj/game/core/runs.clj

```diff
@@ -4,7 +4,7 @@
          update-ice-in-server update-all-ice get-agenda-points get-remote-names
          card-name can-access-loud can-steal?  prevent-jack-out card-flag? can-run?
          update-all-agenda-points reset-all-ice no-action make-run encounter-ends
-         pass-ice do-access successful-run)
+         pass-ice do-access successful-run complete-run)
```

Comment:
Instead of forward declaring the function here, how about moving `complete-run` above `successful-run`.

Reply:
Generally yeah. `declare` exists to help out when necessary (mutually recursive functions) but it’s best to keep things “in order”. Helps with hot reloading code too.